app/src/main/java/org/kiwix/kiwixmobile/zim_manager/fileselect_view/ZimFileSelectFragment.kt

```diff
@@ -81,7 +81,7 @@ open class ZimFileSelectFragment : BaseFragment() {
   }
 
   override fun inject(baseActivity: BaseActivity) {
-    (baseActivity as ZimManageActivity).cachedComponent.inject(this)
```

Comment:
you might want to create a cached component in KiwixMainActivity, it is quite wsteful to create a new activityComponent each time we call inject per fragment

Reply:
Did not know what the point of the cachedComponent was, thanks!